flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -378,6 +378,8 @@ under the License.
 
 									<!-- flink-table-runtime-blink dependencies -->
 									<include>org.codehaus.janino:*</include>
+									<include>org.apache.flink:flink-table-code-splitter</include>
+									<include>org.antlr:antlr4-runtime</include>
```

Comment:
Can we just include antlr:antlr4 in `flink-table-code-splitter`? (And modify NOTICE too)